content/sigs/ux/index.adoc

```diff
@@ -41,11 +46,26 @@ Join the link:https://gitter.im/jenkinsci/ux-sig[Gitter room] for user experienc
 === Contributing
 If you want to contribute we recommend that you join the Jenkins UX SIG. You can find details of when the next meetings are planned in the Jenkins Calendar or via the link:https://gitter.im/jenkinsci/ux-sig[jenkinsci/ux-sig Gitter room].
 
-=== Ongoing projects
+[[ongoing-projects]]
+=== Projects
+
+The special interest group drives a number of initiatives within the link:/project/roadmap/[Jenkins roadmap].
+
+==== UI/UX: Jenkins Look and Feel updates
+
 The first UX SIG project is revamping the Jenkins UI look and feel.
-The initial focus will be based on CSS changes.
+The initial focus will be on CSS changes and key Jenkins components: header, footer, side panels, etc.
+See the jira:JENKINS-60919[] EPIC for more details about the scope.
 Once this project has been completed, a more thorough rewrite of the Jenkins UI will be planned.  
 
+==== Jenkins user interface overhaul
+
+Future project which targets a major redesign of the Jenkins user interface. 
+Goals:
```

Comment:
If there is not an empty line preceding this word, won't it wrap onto the preceding line?  If that's intended, then you can ignore this suggestion.
```suggestion

Goals:
```